spark/v3.5/spark-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/CreateOrReplaceTagExec.scala

```diff
@@ -48,7 +48,8 @@ case class CreateOrReplaceTagExec(
           .orNull
 
         Preconditions.checkArgument(snapshotId != null,
-          "Cannot complete create or replace tag operation on %s, main has no snapshot", ident)
+          "Cannot complete create or replace tag operation on %s, main has no snapshot",
```

Comment:
Include catalog name in the error message.

Reply:
if we do this here, then we should align this for creating/replacing branches as well. Additionally, we should check how we do this in other Exec classes to align things properly. That being said, I think it's better to handle this in a separate PR